compiler/qsc_frontend/src/typeck/rules.rs

```diff
@@ -576,7 +576,9 @@ impl<'a> Context<'a> {
             None => match self.names.get(path.id) {
                 None => converge(Ty::Err),
                 Some(Res::Item(item, _)) => {
-                    let scheme = self.globals.get(item).expect("item should have scheme");
+                    let Some(scheme) = self.globals.get(item) else {
```

Comment:
Could this be covered by a unit test, or does that stop working when #1761 merged because the behavior of the repro changes?

Reply:
I originally had a unit test for it, and then backed it out because the behavior was about to change. But good point, it can still be covered, and you're the one who will have to resolve the conflict 😉

Reply:
added in https://github.com/microsoft/qsharp/pull/1766/commits/da9854d2748022260314891cc635fb59ea7ea983